generators/gae/index.js

```diff
@@ -736,14 +738,15 @@ module.exports = class extends BaseGenerator {
             },
 
             addMavenPlugin() {
+                if (this.abort) return;
                 if (this.buildTool === 'maven') {
                     if (this.gaeCloudSQLInstanceNeeded === 'Y') {
                         this.render('pom-plugin.xml.ejs', rendered => {
                             this.addMavenPlugin('com.google.cloud.tools', 'appengine-maven-plugin', '2.1.0', rendered.trim());
                         });
                     }
                     this.render('pom-profile.xml.ejs', rendered => {
-                        this.addMavenProfile('prod-gae', `            ${rendered.trim()}`);
+                        this.addMavenProfile('prod-gae', `${rendered.trim()}`);
```

Comment:
i think the spaces were added originally to make sure the plugin is properly aligned/indented in the pom.xml. is that still the case if we remove the spacing here?

Reply:
Yes, you are correct. I thought it was just an unnecessary space. 😄 I've added it back.